app/gui/src/controller/graph.rs

```diff
@@ -246,7 +246,7 @@ impl NodeTrees {
         use ast::crumbs::Crumb::Infix;
         // If we have macros in the expression, we need to skip their crumbs, as [`SKIP`] and
         // [`FREEZE`] macros are not displayed in the expression.
-        let skip_macros = self.macros_info.macros_count();
+        let crumbs_to_skip = self.ast_info.ast_crumbs_to_skip();
```

Comment:
The comment still does not mention context switching.

Also, the value may be not the final number crumbs to skip (as we sometimes add 1 for the binding infix). Let it be something like "expression_crumbs_to_skip".

Reply:
Fixed